src/requests/getImages.js

```diff
@@ -0,0 +1,50 @@
+import axios from "axios";
+
+const getImages = (query) => {
+  if(!query) {
+    return Promise.resolve([])
+  } else {
+    return axios
+      .get(`https://images-api.nasa.gov/search?q=${query}`)
+      .then((response) => {
+        const imageResults = (response.data.collection.items)
+
+// ORIGINAL PARSE RESULTS METHOD         
+//
+//        const filteredResults = imageResults.filter(result => result.data[0].media_type === "image")
+//        console.log(imageResults)
+//        const images = filteredResults.map(result => result.links[0].href)
+
+//        return images
+
+        let results=[]
```

Comment:
Could this be a const? Since you arent directly reassigning it theres no need to use the let keyword here

Reply:
I declared results using let as the array would be written to later in the code. Not necessary though as const is better to use